examples/index.md

```diff
@@ -325,54 +289,41 @@ HTTP/1.1 422 Unprocessable Entity
 Content-Type: application/vnd.api+json
 
 {
+  "jsonapi": { "version": "1.0" },
```

Comment:
Here, I brought the `json-api` member into this example to merge it with the one below, since they were both showing more-or-less the same thing.